curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-a-expense-tracker/657e6231622cd919e2d7094b.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 657e6231622cd919e2d7094b
+title: Step 4
+challengeType: 20
+dashedName: step-4
+---
+
+# --description--
+
+Now it's time to be able to append each expense as a dictionary to the `expenses` list. This illustrates how to create functions that manipulate lists by adding new items.
```

Comment:
```suggestion
Now it's time to be able to append each expense as a dictionary to the `expenses` list. "Appending" means adding an item to the end of the list. 
```

Reply:
I would suggest adding a brief context on what "appending" means if this is not mentioned in previous steps.

Reply:
I will commit your suggested change as I think it briefly explains "Appending" without making the step too long.